ui/admin/setup-edit.php

```diff
@@ -306,7 +306,7 @@
                 }
             ?>
             <tr class="no-items<?php echo esc_attr( 1 < $pods_i ? ' hidden' : '' ); ?>">
-                <td class="colspanchange" colspan="4">No fields have been added yet</td>
+                <td class="colspanchange" colspan="4"><?php _e( 'No fields have been added yet', 'pods' ); ?></td>
```

Comment:
Please use `esc_html_e()` instead of `_e()` for echoing values that do not require HTML.